packages/wdio-utils/src/initialisePlugin.ts

```diff
@@ -7,7 +7,7 @@ import { safeRequire } from './utils'
  * 2. otherwise try to require "@wdio/<name>-<type>"
  * 3. otherwise try to require "wdio-<name>-<type>"
  */
-export default function initialisePlugin (name, type) {
+export default function initialisePlugin (name: string, type?: string) {
```

Comment:
add `return X as WebdriverIO.ServiceClass` in the return statements here so you don't need to cast it everywhere else in the code